### DISCUSSION TITLE
rife的官方模型如何替换

### DISCUSSION THREAD

**Discussion by OrangeYoon** [hooke007/mpv_PlayKit#482] [mpv 通用]
![image](https://github.com/hooke007/MPV_lazy/assets/169504647/2540173c-7634-4d54-b93c-a27490193b32)
官方模型是这种格式的，我该怎么转成onnx格式并增加现有的模型呢？砖家们

**Comment by hooke007** [comment]
这里用的就是官模，有什么问题

**Reply by OrangeYoon** [reply]
想换版本更高的17 18，需要转换成onnx吗

**Reply by hooke007** [reply]
https://github.com/AmusementClub/vs-mlrt/releases/tag/external-models
